### PR TITLE
Balance Pass (iteration 1)

### LINKED ISSUE #607: First Balance Pass
After #606, go through the values in each file and check that they are proper.

Additionally, balance the damage and cooldowns of skills, pending playtest.

### PR DESCRIPTION
**Pull request by marco-paulucci**
Closes #607 

Characters
- reduced high variance in character sizes & velocities to prevent imbalances between classes
- body sizes slightly increased
- characters speeds slightly decreased
- both of above changes will make projectile collision function better

Gamemode
- it takes less time for circle to activate
- circle shrinks slower
- minimum circle size reduced

Skills
- Various tweaks. This is a first iteration.

### CODE REVIEW

**Review by mgrunwaldt — DISMISSED** [review]
Need to test it with players but for a first iteration its perfect

File: client/Assets/StreamingAssets/Skills.json [hunk 1]
```diff
@@ -237,9 +237,9 @@
     {
       "Name": "Neon Crash",
       "Cooldown": "5000",
-      "Damage": "2",
+      "Damage": "10",
       "Duration": "300",
-      "SkillRange": "200",
+      "SkillRange": "0",
```
[the remark this answers is not in the document]
Comment (marco-paulucci) [reply client/Assets/StreamingAssets/Skills.json]: thanks for the catch, fixed

**Review by BertovDev — APPROVED** [review]

**Review by manucamejo — APPROVED** [review]

File: client/Assets/StreamingAssets/GameSettings.json [hunk 2]
```diff
@@ -4,9 +4,9 @@
   "board_height": "10000",
   "server_tickrate_ms": "30",
   "game_timeout_ms": "1200000",
-  "map_shrink_wait_ms": "60000",
-  "map_shrink_interval": "25",
-  "map_shrink_minimum_radius": "2000",
+  "map_shrink_wait_ms": "3000",
```
Comment (BertovDev) [inline client/Assets/StreamingAssets/GameSettings.json:7]: is this correct ? 3 seconds

Comment (marco-paulucci) [reply client/Assets/StreamingAssets/GameSettings.json]: it was not, thank you for noticing. fixed!

**Review by tvillegas98 — APPROVED** [review]
Perfectly balanced 😄

**Pull request merged**